ui-modules/login/app/views/main/main.controller.js

```diff
@@ -47,18 +47,28 @@ export function loginStateController($scope, $http, $window, brBrandInfo) {
     $scope.$emit(HIDE_INTERSTITIAL_SPINNER_EVENT);
     $scope.getBrandedText = brBrandInfo.getBrandedText;
 
+    let loginController = this;
+    loginController.error = {};
+
+    var testAuthReq = {
+        method: 'HEAD',
+        url: '/v1/server/up/extended'
+    }
+
+    // If the user is already logged in then redirect to home
+    $http(testAuthReq).then( () => {$window.location.href = '/';} );
```

Comment:
I think that there is a more angularjs way to redirect but I'm not sure I understand what it is.

Reply:
There would be @duncangrant if within the same UI module. But you want to redirect to home, i.e. `/` so this is the right thing to do